contrib/signet/getcoins.py

```diff
@@ -49,4 +49,16 @@
 except:
     print('Unexpected error when contacting faucet:', sys.exc_info()[0])
     exit()
-print(res.text)
+
+# Display the output as per the returned status code
+if res:
+    # When the return code in between 200 and 400 i.e. successful
+    print(res.text)
+elif res.status_code == 400:
```

Comment:
The faucet should return ["429 Too Many Requests"](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/429) which indicates rate-limiting. Returning "400 Bad Request" is wrong. [Could be changed here](https://github.com/kallewoof/bitcoin-faucet/blob/8575d3dc38341561551635793e892be913e3a27d/src/index.ts#L188).

Reply:
The faucet returns "400 Bad Request" in [multiple places](https://github.com/kallewoof/bitcoin-faucet/blob/8575d3dc38341561551635793e892be913e3a27d/src/index.ts). So status 400 doesn't have to be rate limiting. Better not match on it.